lighthouse-core/audits/is-on-https.js

```diff
@@ -74,18 +82,42 @@ class HTTPS extends Audit {
           .filter(record => !HTTPS.isSecureRecord(record))
           .map(record => URL.elideDataURI(record.url));
 
+      /** @type {Array<{url: string, resolution?: string}>}  */
       const items = Array.from(new Set(insecureURLs)).map(url => ({url}));
 
-      let displayValue = '';
-      if (items.length > 0) {
-        displayValue = str_(UIStrings.displayValue, {itemCount: items.length});
-      }
-
       /** @type {LH.Audit.Details.Table['headings']} */
       const headings = [
         {key: 'url', itemType: 'url', text: str_(UIStrings.columnInsecureURL)},
       ];
 
+      // Mixed-content issues aren't emitted until M84.
+      if (artifacts.InspectorIssues.mixedContent.length) {
+        headings.push(
```

Comment:
so does anyone hate this

Reply:
That's my expectation.

Also, I'll file an issue to follow up and remove the `isSecureRecord` part completely once M84 lands.

> Would it be terrible to give every entry a default resolution: undefined then

sure.

Reply:
Also, I'll file an issue to follow up and ~~remove the isSecureRecord part completely~~ make the header always added once M84 lands.

Reply:
cc #10976 some other m84 stuff.

Reply:
bump on

>  I'll file an issue to follow up and make the header always added once M84 lands.

so it's not lost in the middle of this review.

Reply:
https://github.com/GoogleChrome/lighthouse/issues/11076